guides/installation/setups/index.md

```diff
@@ -0,0 +1,44 @@
+---
+nav:
+  title: Setups
+  position: 3000
+
+---
+
+# Setups
+
+## Set up your own environment
+
+Head over to the [Requirements](../requirements) section to install and configure the necessary services like a database and a webserver to a Unix system like Linux, macOS, WSL etc.
+
+## Development setup
+
+::: info
+Technically there is no real difference between a Development Setup and a Production Setup, they only differ on performance and security optimizations.
+:::
+
+> \* These setups are provided and officially supported by shopware AG.
```

Comment:
I would flip this and do the `*` asterisks on not officially supported things. Asterisks always imply a caveat.

Reply:
see https://github.com/shopware/docs/pull/1746#discussion_r2043729535